RockLib.Encryption.Tests/MultiFactory.json

```diff
@@ -10,12 +10,7 @@
                                 "Name": "Default",
                                 "Algorithm": "Aes",
                                 "IVSize": "16",
-                                "Key": {
-                                    "Type": "RockLib.DataProtection.UnprotectedBase64Value, RockLib.DataProtection",
```

Comment:
I thought we got rid of the DataProtection stuff. Did we forget to update the examples?

Reply:
Ah, it's the tests that didn't get updated.